docker/focal-ghc-cabal-stack/Dockerfile

```diff
@@ -38,4 +38,4 @@ RUN  apt-get update \
 
 # Compression utils
 RUN  apt-get update \
-  && apt-get -y install tar xz-utils zip
+  && apt-get -y install tar xz-utils zip zstd
```

Comment:
Do you still use `xz-utils` now that you're using `zstd`?

Reply:
Don't think so! *yeet*